clients/drcachesim/CMakeLists.txt

```diff
@@ -136,6 +136,7 @@ endif()
 
```

Comment:
There are drmemtrace features that rely on the .zip archive format, such as fast skipping over large instruction counts, which is in turn a key part of the simulator (and regular analysis tool) interface for regions of interest or simpoints.  Such fast skipping is not possible with a single-encrypted file like this lz4 format so we are not excited about promoting it and are not likely to use it ourselves.  We may also continue adding features that require zip.

Have you tried Z_BEST_SPEED instead of Z_DEFAULT_COMPRESSION?

Reply:
If your use cases don't need fast skipping, lz4 may well work well.  A chunked format using lz4 is possible but not as easy as zip.  If you could make lz4 not be the default in this PR (we'd prefer the skip features to be there by default) and require an option to turn on that sounds good.